.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,13 @@ Images or gifs: Include before and after screenshots or gifs/videos when it make
 
 Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?
 
+(remove me) Please follow the following guidelines when writing testing instructions:
+
+- If there is no similar flow in the [Critical flows](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Critical-flows) include screenshots.
```

Comment:
I'm no native english speaker but here a "," ("Critical flows,") would improve the spelling I believe.

Reply:
Good point, I'll change this to "Include screenshots if there is no similar flow in the Critical flows" 🙂

Reply:
Changed in c7a47eb8544c1c55dbb269cd9f1a6af2ce18eb5c,  let me know what you think!

Reply:
Looks good!